packages/manager/modules/dbaas-logs/src/logs/empty/empty.controller.js

```diff
@@ -0,0 +1,11 @@
+import placeholder from '../../assets/placeholder.png';
+
+export default class {
+  constructor() {
+    this.placeholder = placeholder;
```

Comment:
do you forget to inject a placeholder? looks like it's not defined.

Reply:
i import placeholder on line 1, isn't the right way ? however, i can remove placeholder state on this part of code because it can be useless on close scope of this feature :)

Reply:
however, this code work correctly ^^